benchmarks/python/conftest.py

```diff
@@ -39,14 +53,21 @@ def disable_benchmarking(request):
     return request.config.getoption("--disable-benchmarking")
 
 
-def pytest_make_parametrize_id(val):
-    return repr(val)
+def pytest_make_parametrize_id(val, argname):
+    if isinstance(val, tuple):
+        return f'{argname}=[{"_".join(str(v) for v in val)}]'
+    return f"{argname}={repr(val)}"
```

Comment:
Cool. I didn't know about this hook. In the future we might want to extend it further but this looks good to me for now.